### PR TITLE
DSP-21668 Bump version of shiro (dse-2.2)

### PR DESCRIPTION
**Pull request by brijeshdungarakoti**

### CODE REVIEW

**Review by justinchuch — COMMENTED** [review]
When creating a PR, please have the JIRA ticket number in the title:

For example:
```
DSP-21668: Bump shiro version to 1.7.0
```

When writing the commit message, please use the format:

```
<JIRA ticket number>: <JIRA ticket title or description of the changes>
```

For example:
```
DSP-21668: Bump shiro version to 1.7.0
```

The PR title and the commit message can be the same.

May I know if you have run any tests with the change?

Thanks!

File: project/Dependencies.scala [hunk 1]
```diff
@@ -82,7 +82,8 @@ object Dependencies {
   )
 
   lazy val securityDeps = Seq(
-    "org.apache.shiro" % "shiro-core" % shiro
+    "org.apache.shiro" % "shiro-core" % shiro,
+    "commons-beanutils" % "commons-beanutils" % "1.9.4" exclude("commons-beanutils","commons-beanutils")
```
Comment (justinchuch) [inline project/Dependencies.scala]: Question: may I know why we need to exclude `commons-beanutils`?

Comment (brijeshdungarakoti) [reply project/Dependencies.scala]: Hi Justin,
If not excluding other common-beanutil version,It is downloading all other previous versions and build was getting failed with 
the exception related to beanutil(http://jenkins2.datastax.lan:8080/view/All/job/DSE_Spark_Jobserver_2/235/console).
Including 1.9.4 version and discarding lower version resolved that error and build was successful.

Comment (justinchuch) [reply project/Dependencies.scala]: Could you please show the dependency tree and which libs are using `common-beanutils`?

**Review by justinchuch — APPROVED** [review]
LGTM: +1, as long as the tests in spark-joberver and bdp passes.

**Pull request merged**